lib/concentrate/vehicle_position.ex

```diff
@@ -0,0 +1,79 @@
+defmodule Concentrate.VehiclePosition do
+  @moduledoc """
+  Structure for representing a transit vehicle's position.
+  """
+  import Concentrate.StructHelpers
+
+  defstruct_accessors([
+    :id,
+    :trip_id,
+    :stop_id,
+    :label,
+    :license_plate,
+    :latitude,
+    :longitude,
+    :bearing,
+    :speed,
+    :odometer,
+    :stop_sequence,
+    :block_id,
+    :operator_id,
+    :operator_name,
+    :run_id,
+    :last_updated,
+    status: :IN_TRANSIT_TO
+  ])
```

Comment:
This looks a lot like `Realtime.Vehicle`. Could we get by with removing one of the definitions?

Reply:
That would mean changing the structure of the client data. We can discuss whether or not we want to do that, but that seems like too much for this piece of work.